client/src/api/users.js

```diff
@@ -15,14 +15,15 @@ export const createOrLoginUser = async (method, userData) => {
     if (response.data.hasOwnProperty('err')) return { ...response.data };
 
     localStorage.setItem('jwtToken', response.data.token);
-
+    console.log(response.data.user.notificationCount)
```

Comment:
Should probably get rid of any console logs before merging

Reply:
thanks. just removed it.